pkg/platform/errors.go

```diff
@@ -0,0 +1,13 @@
+package platform
+
+import (
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/rollbar"
+)
+
+type ErrCountryBlocked struct{ *locale.LocalizedError }
+
+func NewCountryBlockedError() *ErrCountryBlocked {
+	rollbar.DoNotReportMessages.Add(locale.T("err_country_blocked"))
+	return &ErrCountryBlocked{LocalizedError: locale.NewInputError("err_country_blocked")}
+}
```

Comment:
This was moved from it's original location so I could import `retryhttp` in the `api` package without an import cycle.

Reply:
Are you sure this is still relevant? I don't see this being called from `retryhttp`.

Reply:
It's being used in a different file but same package: https://github.com/ActiveState/cli/blob/DX-1787/internal/retryhttp/roundtripper.go#L33